src/styles/blocks/head-block/icon.scss

```diff
@@ -0,0 +1,7 @@
+.icon {
```

Comment:
There are no data-qa attributes applied. These attributes are important for testing. Make sure to add data-qa attributes to your elements to facilitate testing.